cwms-data-api/src/main/java/cwms/cda/api/LocationGroupController.java

```diff
@@ -227,10 +227,9 @@ public void create(@NotNull Context ctx) {
             if (!deserialize.getLocationCategory().getOfficeId().equalsIgnoreCase(CWMS_OFFICE)
                     && (!deserialize.getOfficeId().equalsIgnoreCase(deserialize.getLocationCategory().getOfficeId())
                     || deserialize.getOfficeId().equalsIgnoreCase(CWMS_OFFICE))) {
-                CdaError re = new CdaError("Office ID cannot be CWMS and must match the location category office ID");
-                logger.info(() -> re + System.lineSeparator() + "for request " + ctx.fullUrl());
-                ctx.status(HttpServletResponse.SC_BAD_REQUEST).json(re);
-                return;
+                CdaError re = new CdaError("Location Group office ID cannot be CWMS and must match the "
+                    + "Location Category office ID");
+                throw new IllegalArgumentException(re.toString());
```

Comment:
just pass in the message, no need to create an exception just to call toString() on it

Reply:
Removed CdaError